web_app/src/personalBoard.js

```diff
@@ -194,17 +210,48 @@ function PersonalBoard(props) {
         }
     }
 
+    const play = () => {
+        setState(prevState => {return {...prevState, open: true}});
+       console.log(state.open)
+    }
+    //const classes = useStyles();
+
+    const stop = () => {
+        setState(prevState => {return {...prevState, open: false}});
+    }
+
     return (
         state.board !== null ? 
             <div style={{display: 'flex', flexDirection: 'column', padding: "20px"}} >
-                {
-                    <h2>{state.board.boardName}</h2>
-                }
+                <AppBar color="inherited" position="static">
+                    <Toolbar variant="dense">
+                        
+                        { <h1 style={{marginRight:'40px', flexGrow: '1', fontFamily: 'Arial'}}>{state.board.boardName}</h1> }
+                        <Button 
+                            variant="contained" 
+                            style={{backgroundColor: '#4CAF50', width:'100px', padding: '5px 5px 5px 4px',marginRight: '20px'}} 
+                            onClick={play}
+                            startIcon={<PlayArrowIcon />}>
+                            Play  
+                        </Button>
+                        {/* follow related */}
+                        <Link style={{marginRight:'20px'}} to={`/pboards/followers/${userid}/${id}`}>
+                            <h3>{`Followers: ${state.followers.length}`}</h3>
+                        </Link>
+
+                    </Toolbar>
+                </AppBar>
+                {/* Playlist modal */}
+                { state.open ? (
+                        <div>
+                         <PlayQueue 
+                            queue={state.board.queue}
+                            stop={stop}
+                            open={state.open}
```

Comment:
just FYI you don't need it inside the PlayQueue, then probably don't give it as a prop

Reply:
the 'open' prop